kolibri/plugins/epub_viewer/assets/src/views/SettingsSideBar.vue

```diff
@@ -1,6 +1,17 @@
 <template>
 
-  <SideBar>
+  <SideBar :class="['epub-sidebar', getClassByWindowSize]">
+    <div style="display: flex; flex-direction: row; justify-content: space-between;">
```

Comment:
We do sometimes use style like this, but for these, which are relatively straightforward and are not dependent on things like computed values, let's put them in a class. If there's only one value (i.e. "padding: 0"), it's okay to leave it as an inline `style`, but I think once we get to three or so items here, it's easier to read if it's a class.

Reply:
done